### ISSUE TITLE
no-empty-blocks false positive in inherited constructor

### ISSUE THREAD

**Issue by quezak** [protofire/solhint#242]
Hello! I'm migrating from ethlint and found the same issue that I [reported there](https://github.com/duaraghav8/Ethlint/issues/264) some time ago :)

In an inherited contract, the constructor often just calls the base constructor, and its block is empty. This should be an exception from the `no-empty-block` rule, but now it causes a `warning    Code contains empty block`.

Copied from the [official example](https://solidity.readthedocs.io/en/v0.7.0/contracts.html#arguments-for-base-constructors).
```solidity
contract Base {
    uint x;
    constructor(uint _x) { x = _x; }
}

// or through a "modifier" of the derived constructor.
contract Derived2 is Base {
    constructor(uint _y) Base(_y * _y) {}
}
```

**Comment by fvictorio** [comment]
Yes, this makes a lot of sense! Should be easy to implement. I can give you some pointers if you feel like giving this a shot. If not, I'll try to fix it soon.

**Comment by emretepedev** [comment]
same issue here.

**Comment by Janther** [comment]
Ethlint did address this issue duaraghav8/Ethlint/issues/264

**Comment by johnnyshankman** [comment]
Same here as well.

**Comment by sebastianst** [comment]
@fvictorio any progress here?

**Comment by dbale-altoros** [comment]
Close with 
https://github.com/protofire/solhint/pull/418
